Gemfile

```diff
@@ -1,8 +1,9 @@
 source 'https://rubygems.org'
 
-# Specify your gem's dependencies in bitreserve.gemspec
 gemspec
 
+gem 'pry'
```

Comment:
Since we're adding this, how about making `rake console` use pry instead of IRB as well?